wvm-apps/wvm-exexed/crates/reth-exexed/src/constant.rs

```diff
@@ -0,0 +1,2 @@
+pub const WVM_DEVNET_VERSION: &str = "v0.2.0";
+pub const WVM_ALPHANET_VERSION: &str = "v0.1.0";
```

Comment:
alphanet is at v0.1.1

Reply:
And lets add WVM_OWNER_WALLET to this file, as array of "owners" maybe? And lets name that array something more meaningful like `DATA_PUBLISHERS`